qrest/conf.py

```diff
@@ -41,6 +42,7 @@ def __init__(
         default: Optional[str] = None,
         choices: Optional[list] = None,
         description: Optional[str] = None,
+        example: Optional = None,
```

Comment:
`Optional` without type, is that desirable. I'm not even sure it's allowed.

Reply:
The type of example isn't fixed, as it depends on what type of value is expected by the endpoint.